apps/storybook.namekit.io/stories/Namekit/Tooltip.stories.tsx

```diff
@@ -45,6 +71,28 @@ export const LeftPlacement: Story = {
     trigger: <>Tooltip Trigger</>,
     children: <>Tooltip content</>,
   },
+  render: (args) => {
+    return (
+      <div className="nk-w-full nk-flex nk-justify-center">
+        <Tooltip {...args} />
+      </div>
+    );
+  },
+};
+
+export const TopPlacement: Story = {
+  args: {
+    placement: "top",
+    trigger: <>Tooltip Trigger</>,
+    children: <>Tooltip content</>,
+  },
+  render: (args) => {
+    return (
+      <div className="nk-h-screen nk-flex nk-items-center">
+        <Tooltip {...args} />
+      </div>
+    );
+  },
 };
 
 export const DefaultMaxTooltipWidth: Story = {
```

Comment:
Adding a comment here that GitHub won't let me add several lines lower where I really want to comment.

![CleanShot 2024-09-17 at 16 53 23](https://github.com/user-attachments/assets/7ce73c7f-2709-4cdb-91ce-a40184e3d6a7)

This creates a maintenance burden because it's easy to forget to update the storybook story if we update this constant later. Better instead to write this as: "This is the default max tooltip width. You can customize the max width if desired. See the "Custom Max Tooltip Width" story for an example of setting a custom max width."

![CleanShot 2024-09-17 at 16 56 00](https://github.com/user-attachments/assets/c24fef3f-94fd-48bd-82bc-6fdd40937493)

There's no benefit to changing the trigger just for this story (relative to all the others). Better to just keep this the same as all the other stories: "Tooltip Trigger".

![CleanShot 2024-09-17 at 16 57 29](https://github.com/user-attachments/assets/a702be10-ed2c-4e86-8c95-51264667f336)

It's not good to write content in our stories like this. The problem is that we need to consider how users can customize the control for the `maxTooltipWidth`. Therefore, don't make explicit reference in your writing to a specific value such as "600px" that the user might override through the storybook controls. If you write in this way it can be a source of confusion. You need to think from the perspective of people using and interacting with these stories.

Here's a better way to write this: "This is a customized max tooltip width. You can customize the max width if desired." There's no need to write other ideas here.